### ISSUE TITLE
run_ner.py of huggingface

### ISSUE THREAD

**Issue by vthost** [dmis-lab/biobert#66]
Hi,
I want to run a pytorch version and managed to run the code from SolvingAlmostAnythingWithBert, but somehow only for some datasets, and the results seem to be strange. I also tried the huggingface code (adapting the dataset names, special symbols), but there I get very strange results as well:
f1 = 0
loss = 7.114976575394394e-05
precision = 0
recall = 0
Do you maybe now if I can use that code at all or what I would have to adapt apart from the things I mentioned? 
Thank you already!

**Comment by vthost** [comment]
Just for the records...
I found the problem: in the data files the "B" is separated by \t and this is not recognized in the utils_ner of huggingface, so none of those "B"s was recorded. But that's an easy fix, now it runs :)
